src/components/common/FileUploadPdf.vue

```diff
@@ -0,0 +1,316 @@
+<template>
```

Comment:
Is this the same as the component in Filings UI, or are there changes (that maybe should be back-ported)?

Reply:
Should be the same, if it isn't the same - I'll make it consistent with Filings UI. 

This component saved me quite a bit of time, it's much better than the implementation in `business-create-ui`.

Reply:
I noticed it's a bit different (something related to page size + the way the pdfjs-dist is loaded). Please make it the same (one way or another) or create a new ticket to do so. I don't want to overload you right now but I do want to keep track of tech debt.

Reply:
Should be very similar now

Reply:
Will be handled in bcgov/entity#16489.

Reply:
> Should be very similar now - except for the loading on created

Much appreciated!